marimo/_tutorials/intro.py

```diff
@@ -164,15 +186,16 @@ def __(mo):
 @app.cell(hide_code=True)
 def __(mo):
     mo.md(
-        """## 2. UI elements
+        """
+        ## 2. UI elements
 
-        Cells can output interactive UI elements. Interacting with a UI
-        element **automatically triggers notebook execution**: when
-        you interact with a UI element, its value is sent back to Python, and
-        every cell that references that element is re-run.
+            Cells can output interactive UI elements. Interacting with a UI
+            element **automatically triggers notebook execution**: when
+            you interact with a UI element, its value is sent back to Python, and
+            every cell that references that element is re-run.
 
-        marimo provides a library of UI elements to choose from under
-        `marimo.ui`.
+            marimo provides a library of UI elements to choose from under
+            `marimo.ui`.
```

Comment:
@mscolnick Is it possible the view as markdown toggle is adding these indents?

Reply:
seems likely - could be because the original was on the same line. 

this shouldnt happen if we always start in markdown - and only would if the user moves text to the same line as the opening `"""`